web/browser.py

```diff
@@ -105,66 +103,113 @@ def open(self, url, data=None, headers={}):
 
     def show(self):
         """Opens the current page in real web browser."""
-        f = open('page.html', 'w')
+        f = open("page.html", "w")
         f.write(self.data)
         f.close()
 
-        url = 'file://' + os.path.abspath('page.html')
+        url = "file://" + os.path.abspath("page.html")
         webbrowser.open(url)
 
     def get_response(self):
         """Returns a copy of the current response."""
-        return addinfourl(BytesIO(self.data), self._response.info(), self._response.geturl())
+        return addinfourl(
+            BytesIO(self.data), self._response.info(), self._response.geturl()
+        )
 
     def get_soup(self):
         """Returns beautiful soup of the current document."""
         import BeautifulSoup
+
         return BeautifulSoup.BeautifulSoup(self.data)
 
     def get_text(self, e=None):
         """Returns content of e or the current document as plain text."""
         e = e or self.get_soup()
-        return ''.join([htmlunquote(c) for c in e.recursiveChildGenerator()
-                       if isinstance(c, text_type)])
+        return "".join(
+            [
+                htmlunquote(c)
+                for c in e.recursiveChildGenerator()
+                if isinstance(c, text_type)
+            ]
+        )
```

Comment:
This is a case where i find the initial version is more readable.

Reply:
It could be more compact, but i'm ok with this new format, more (reasonable) space and more readable.